conceptnet5/wiktparse/rules.py

```diff
@@ -366,7 +372,7 @@ def parse_structured_entry(self, structure):
                 print(traceback.format_exc())
                 failures += 1
 
-        assert failures <= 1
+        assert failures <= 1 # Why is 1 failure ok?
```

Comment:
On the topic of "Why is 1 failure ok?": if you can get it to run with 0 failures, by all means do so.

The unpredictability of what Wiktionary throws at us has made this unachievable so far, but if a strangely-formatted entry can just quietly produce no assertions instead of breaking the parser, that would be great.

The main concern would be that a change to the code might make the failures disappear because they're "succeeding" in producing nonsense assertions. This is why unit tests are nice to have, though.